Docs/source/building/summit.rst

```diff
@@ -130,11 +130,14 @@ Optionally, download and install :ref:`libEnsemble <libensemble>` for dynamic en
    python3 -m pip install --upgrade mpi4py --no-binary mpi4py
    python3 -m pip install --upgrade -r $HOME/src/warpx/Tools/LibEnsemble/requirements.txt
 
-Then, ``cd`` into the directory ``$HOME/src/warpx`` and use the following commands to compile:
+Then, ``cd`` into the directory ``$HOME/src/warpx`` and use the following commands to compile with :ref:`_building-cmake`:
```

Comment:
Sorry for this, I meant the link below the command :)
```suggestion
Then, ``cd`` into the directory ``$HOME/src/warpx`` and use the following commands to compile:
```

Reply:
Thanks @ax3l
Oh no need to be sorry! It makes more sense that way.